### PR TITLE
Add 5.4.0 benchmark

### PR DESCRIPTION
**Pull request by Yui-Song**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: benchmark/v5.4-performance-benchmarking-with-tpcc.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+   balance roundrobin                      # 连接数最少的服务器优先接收连接。`leastconn` 建议用于长会话服务，例如 LDAP、SQL、TSE 等，而不是短会话协议，如 HTTP。该算法是动态的，对于启动慢的服务器，服务器权重会在运行中作调整。
+   server tidb-1 10.9.18.229:4000 check inter 2000 rise 2 fall 3       # 检测 4000 端口，检测频率为每 2000 毫秒一次。如果 2 次检测为成功，则认为服务器可用；如果 3 次检测为失败，则认为服务器不可用。
+   server tidb-2 10.9.39.208:4000 check inter 2000 rise 2 fall 3
+   server tidb-3 10.9.64.166:4000 check inter 2000 rise 2 fall 3
+```
+
+## 测试方案
+
+1. 通过 TiUP 部署 TiDB v5.4.0 和 v5.3.0。
+2. 创建数据库 tpcc：`create database tpcc;`
```
Comment (TomShawn) [inline benchmark/v5.4-performance-benchmarking-with-tpcc.md:114]:
```suggestion
2. 创建数据库 tpcc：`create database tpcc;`。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by dbsid — APPROVED** [review]
/lgtm

**Pull request merged**